core/src/blocktree_processor.rs

```diff
@@ -193,11 +187,11 @@ pub fn process_blocktree(
                 bank_id: slot,
                 entry_height: starting_entry_height,
             };
-            fork_info.push((starting_bank, bfi));
+            fork_info.push((bank, bfi));
```

Comment:
This doesn't seem to unwind the entries from the partial slot.  I thought I wrote a test that would go red if we tried this

Reply:
Ah, this test would have probably failed with this line:
https://github.com/solana-labs/solana/blob/6cf6a1ccc3f05786692b5dab7ce0b98d5a65adbb/tests/multinode.rs#L48
But it's deleted now so problem solved

Reply:
roger, will fix

Reply:
this test test_process_blocktree_with_incomplete_slot should have covered it

still looking